### DISCUSSION TITLE
Backspace and Delete key behavior in PCManFM

### DISCUSSION THREAD

**Discussion by frogsperg** [lxqt/lxqt#2446] [Q&A]
All of a sudden I can't edit file names in the file manager with the backspace or delete keys. In QTerminal the delete key returns the _tilde_ character, but backspace functions properly. Also, in QTerminal my arrow keys only return A, B, C, and D. I cannot arrow up or down to scroll through previous commands. I cannot arrow left or right to move the cursor back and forth in a line of text.

I've tried editing and replacing bash and input runcom files. I've reinstalled QTerminal. In Termit the keys function just fine.

echo $SHELL
/bin/bash

echo $TERM
xterm-256color

What else can I do?

**Comment by tsujan** [comment]
> All of a sudden...

If it happened to me, I'd check what happened before this change of behavior. Did I upgrade or install something? Did I change a basic setting or config file? I'd also check if it was specific to some kind of apps, or whether it would happen to a new user, etc.

**Reply by frogsperg** [reply]
Well that's just it. I didn't make any changes to anything. I just noticed that I couldn't use my arrow keys in qterminal, while I was using qterminal. There was a tech Teamviewing my screen while I was troubleshooting a switch. It was "all of a sudden" and he noticed it as well. We were both caught off guard and said "What happened?" After I got back into the office I noticed the backspace and delete behavior had changed in pcmanfm. So far, those are the only two apps I've experience these problems.

EDIT: Updated the system and tested another user profile. Same results.

**Reply by tsujan** [reply]
> There was a tech Teamviewing my screen

Teamviewer? Can't it affect input?

EDIT: I've never installed it, but it seems to have a daemon.

**Reply by frogsperg** [reply]
I thought the same thing, but so far haven't come across any input setting that I can change to fix it.

**Reply by stefonarch** [reply]
Did you try adding an user and testing it? To see if it's on system level or personal settings level (I suppose the latter).

**Reply by tsujan** [reply]
If Teamviewer is the cause and has a root service, it could affect new users too.

**Reply by frogsperg** [reply]
I tested with another existing user account and with a newly created account. Same results.

**Reply by stefonarch** [reply]
Did you install/update anything? Which distro?

**Reply by frogsperg** [reply]
Siduction. Nothing other than system updates using _apt upgrade_.

EDIT: On another laptop with Siduction, same updates, no problem.

**Reply by frogsperg** [reply]
I found that changing emulation in qterminal from default to solaris restored arrow key behavior to normal. I still haven't solved the issue in pcmanfm.
